knowledge_graphs/knowledge_graph_validator_composed.py

```
@@ -49,261 +50,120 @@
+            result.summary = self._generate_summary(result)
 
-        Returns:
-            float: The average confidence score across all validations, or 1.0 if no confidence scores are present.
-        """
-        all_confidences = []
-
-        # Collect confidence scores from all validations
-        for validation in result.import_validations:
-            if hasattr(validation.validation, 'confidence'):
-                all_confidences.append(validation.validation.confidence)
+        except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)